ci-operator/step-registry/cucushift/pre/cucushift-pre-commands.sh

```diff
@@ -20,7 +20,7 @@ fi
 users=""
 htpass_file=/tmp/users.htpasswd
 
-for i in $(seq 1 30);
+for i in $(seq 1 50);
```

Comment:
Increase the number of prepared test accounts, so that we have enough test accounts for increased parallel tests